src/main/java/dao/UserDao.java

```diff
@@ -59,6 +63,91 @@ public ResultSet getLoginData(String username){
 		return result;
 	}
 	
+	public boolean writeComment(CommentBean comment) {
+		boolean result = false;
+		String sql = "INSERT INTO public.comment (id_user, id_commentator, comment, vote) VALUES (?, ?, ?, ?)";
+		Connection connection = daoFactory.getConnection();
+		
+		try {
+			PreparedStatement query = connection.prepareStatement(sql);
+			query.setInt(1, comment.getUserId());
+			query.setInt(2, comment.getCommentatorId());
+			query.setString(3, comment.getComment());
+			query.setBoolean(4, comment.getVote());
+			
+			if(query.executeUpdate() != 0) {
+				result = true;
+			}
+			connection.close();
```

Comment:
connection.close() should be in finally block, otherwise if an exception occurs the connection will stay open.